style.css

```diff
@@ -622,3 +625,128 @@ body {
   line-height: 24px;
   color: #000;
 }
+
+/* contact me */
+.contact-form {
```

Comment:
- [OPTIONAL] This is a critical point that should be mentioned in previous reviews, Kindly stick to the Figma design as much as possible and adjust the positioning of the contact form background 💪🏼 

### Your Design
![Screenshot from 2023-04-06 15-09-38](https://user-images.githubusercontent.com/68030297/230403027-5fb1d7b8-9001-47c6-a3b0-a7d0438d7aae.png)

### Figma Design
![Screenshot from 2023-04-06 15-10-23](https://user-images.githubusercontent.com/68030297/230403206-b8b15b0d-a84e-4556-8e37-d0f42f63d0eb.png)